pkg/frontend/openshiftcluster_putorpatch_test.go

```diff
@@ -1119,7 +1119,6 @@ func TestPutOrPatchOpenShiftClusterAdminAPI(t *testing.T) {
 		{
 			name: "patch a none maintenance state cluster with maintenance unplanned request",
 			request: func(oc *admin.OpenShiftCluster) {
-				oc.Properties.MaintenanceTask = admin.MaintenanceTaskEverything
```

Comment:
I'm a little confused by the fact that you had to remove this line to fix the test. To help me understand: what's the conceptual difference between sending an empty PATCH request and sending a PATCH request that includes `MaintenanceTaskEverything`?

Reply:
It doesn't matter. When you don't set the maintenance task, the default is everything: https://github.com/Azure/ARO-RP/blob/9a9edacf6be058d0832c5827e837ecb15a692e17/pkg/api/defaults.go#L37